actions/knowledge/filehelper.ts

```diff
@@ -31,3 +32,20 @@ export async function getFileOrFolderSizeInKB(
 export async function getBasename(filePath: string): Promise<string> {
   return path.basename(filePath);
 }
+
+export async function importFiles(
+  files: string[],
+  type: 'local' | 'notion'
```

Comment:
are there going to be many more types of files to import?

Reply:
Yes, more types as we add more integration